basedtyping/__init__.py

```diff
@@ -24,10 +24,20 @@
     Cannot actually be called unless it's narrowed, so it should only really be used as
     a bound in a ``TypeVar``.
     """
+
+    Never = NoReturn
 else:
     # for isinstance checks
     Function = Callable
 
+    class _NeverMetaclass(type):
+        def __instancecheck__(cls, instance: object) -> bool:
```

Comment:
`__subclasscheck__` as well.

Reply:
wasn't able to use `__subclasscheck__` because it only works the other way around (ie. works on its subtypes which is useless because i banned subtyping) but not its supertypes

so instead i hardcoded it in `issubform`

Reply:
```py
class M(type):
    def __subclasscheck__(self, subclass):
        return True

class N(metaclass=M):
    ...


print(issubclass(int, N))  # True
```
I'm not sure what you mean.